Helper/Api/Url.php

```diff
@@ -371,4 +371,9 @@ public function getClientName($storeId = false, $websiteId = false)
     {
         return $this->getVersionHelper()->getClientName($storeId, $websiteId);
     }
+
+    public function getDefaultClientName($storeId)
+    {
+        return $this->getVersionHelper()->setStoreId($storeId)->getClientName();
```

Comment:
as i wrote in chat, setStoreId is not in current github version
to be sure that specific id is used just pass into config function first param as store id, or second as website id
in magento some class has the default store id as const which you can also bind in and use as first param here in getClientName.

please local test before push ;)